base/fitness_model.py

```diff
@@ -253,9 +282,7 @@ def calc_time_censored_tree_frequencies(self):
         print("fitting time censored tree frequencies")
```

Comment:
this is kind of confusing. there is partial overlap with `calc_node_frequencies`, the results are saved in the same data structure as above, but only the slope is used and that is saved somewhere else with no reference to the `region` qualifier. I think this would benefit from thinking through what exactly this is supposed to do what is needed as output.